StackCalculator/StackCalculator.Tests/StackCalculatorTest.cs

```diff
@@ -0,0 +1,76 @@
+using Calculator.exceptions;
+
+namespace Calculator.Tests;
+
+public class StackCalculatorTest
+{
+    private const double DELTA = 0.001;
+    
+    private bool ApproximatelyEqual(double first, double second)
+    {
+        return first >= second - DELTA && first <= second + DELTA;
+    }
+    
+    private static IEnumerable<TestCaseData> Stacks
+        => new TestCaseData[]
+        {
+            new TestCaseData(new StackArrayImplementation()),
+            new TestCaseData(new StackListImplementation()),
+        };
+    
+    [TestCaseSource(nameof(Stacks))]
+    public void Calculate_shouldReturn2ForSimpleExpression(Stack stack)
+    {
+        // arrange
+        StackCalculator calculator = new StackCalculator(stack);
```

Comment:
```suggestion
        StackCalculator calculator = new(stack);
```
Не пишите в объявлении тип дважды